extensions/wikia/SitemapXml/SitemapXmlController.class.php

```diff
@@ -114,21 +114,33 @@ private function getLastMod( $touched ) {
 		return date( 'Y-m-d\TH:i:s\Z', $touchedTime );
 	}
 
-	private function generateSitemapPage( $namespace, $page ) {
+	private function generateSitemapPage($parsedPath) {
```

Comment:
Here are our code style guidelines for PHP: https://github.com/Wikia/guidelines/blob/master/PHP/CodingConventions.md. I think you need to apply those.